app/src/pages/Play/YourTurnContent.tsx

```diff
@@ -50,6 +51,8 @@ function YourTurnContent(props: {
   activeTurn: CurrentGameSubscription["games"][0]["turns"][0]
   activeTurnPlayState: ActiveTurnPlayState
   secondsLeft: number
+  currentRoundId: Round["id"]
+  nextRoundId?: Round["id"] | undefined
```

Comment:
fyi, this is redundant

```suggestion
  nextRoundId?: Round["id"]
```

is sufficient!